### ISSUE TITLE
Fix scroll up motion on header

### ISSUE THREAD

**Issue by codingjeremy** [android/views-widgets#80]
<a href="https://github.com/mwajeeh"><img src="https://avatars1.githubusercontent.com/u/6572067?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [mwajeeh](https://github.com/mwajeeh)**
_Friday Jul 26, 2019 at 12:01 GMT_
_Originally opened as https://github.com/android/android-ConstraintLayoutExamples/pull/93_

----

Removed useless swipe listener on header which was eating touch and causing parent motion layout to not scroll up if you swipe up in header area.

----

_**[mwajeeh](https://github.com/mwajeeh)** included the following code: https://github.com/android/android-ConstraintLayoutExamples/pull/93/commits_

**Comment by codingjeremy** [comment]
<a href="https://github.com/googlebot"><img src="https://avatars3.githubusercontent.com/u/5384588?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [googlebot](https://github.com/googlebot)**
_Friday Jul 26, 2019 at 12:01 GMT_

----


Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: **Please visit <https://cla.developers.google.com/> to sign.**

Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.

----

#### What to do if you already signed the CLA

##### Individual signers

*   It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).

##### Corporate signers

*   Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google.com/docs/cla/#troubleshoot)).
*   The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
*   The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
		

ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgooglesamples%2Fandroid-ConstraintLayoutExamples%2Fpull%2F93) for more info**.

**Comment by codingjeremy** [comment]
<a href="https://github.com/mwajeeh"><img src="https://avatars1.githubusercontent.com/u/6572067?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [mwajeeh](https://github.com/mwajeeh)**
_Friday Jul 26, 2019 at 12:03 GMT_

----

I signed it!

**Comment by codingjeremy** [comment]
<a href="https://github.com/googlebot"><img src="https://avatars3.githubusercontent.com/u/5384588?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [googlebot](https://github.com/googlebot)**
_Friday Jul 26, 2019 at 12:03 GMT_

----

CLAs look good, thanks!

ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgooglesamples%2Fandroid-ConstraintLayoutExamples%2Fpull%2F93) for more info**.

**Comment by CeH9** [comment]
Why it still isn't merged into master?
